cmd/launcher/uninstall_linux.go

```diff
@@ -0,0 +1,85 @@
+//go:build linux
+// +build linux
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/pkg/errors"
+)
+
+func removeLauncher(ctx context.Context, logger log.Logger, identifier string) error {
+	if identifier == "" {
+		identifier = "kolide-k2"
+	}
+
+	isDebianOS, err := isDebian(ctx, logger)
+	if err != nil {
+		return err
+	}
+
+	serviceName := fmt.Sprintf("launcher.%s", identifier)
+	packageName := fmt.Sprintf("launcher-%s", identifier)
+
+	// Stop launcher service
+	if _, err := tablehelpers.Exec(ctx, logger, 30, []string{"systemctl"}, []string{"stop", serviceName}); err != nil {
+		return err
+	}
+
+	// Disable launcher service
+	if _, err := tablehelpers.Exec(ctx, logger, 30, []string{"systemctl"}, []string{"disable", serviceName}); err != nil {
+		return err
+	}
+
+	// Tell the appropriate package manager to remove launcher
+	if isDebianOS {
```

Comment:
I'd recommend a switch/case style approach here. There's more than just dpkg and rpm, so we'll need a default that errors out